skycatalogs/_version.py

```diff
@@ -1 +1 @@
-__version__ = "1.6.0-rc2"
+__version__ = "1.7.0-rc1"
```

Comment:
This seems premature.

Reply:
I didn't mean to imply that this particular commit is going to be tagged, uploaded to PyPI, etc. 
But there have been significant changes since the last actual release, not just patches. I probably should have advanced the minor version some time ago.